### PR TITLE
Add jobs to CI build

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
PR that adds CI-build jobs to this repo

**How to test: **
- [x] open the travis build checks

**Expected outcome: **
- [x] Several build jobs built 
- [x] Screenshot

- [ ] Reviewed 
- [x] Tested by @patrikgrenfeldt 
- [ ] merge & deploy OK 

This is a minor bump since we add CI func but we don't add any end user func

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
Test results:
image

image
👍

**Comment by patrikgrenfeldt** [comment]
image
This is expected until we merge this PR 👍

**Comment by patrikgrenfeldt** [comment]
@ingkebil do you have any further comments?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,8 @@
 *.pyc
+demux.egg-info/dependency_links.txt
+demux.egg-info/entry_points.txt
+demux.egg-info/not-zip-safe
+demux.egg-info/PKG-INFO
+demux.egg-info/requires.txt
+demux.egg-info/SOURCES.txt
+demux.egg-info/top_level.txt
```
Comment (ingkebil) [inline .gitignore]: ... why not just add demux.egg-info instead of all the file in it?

**Comment by ingkebil** [comment]
I haven't really looked at this. I trust your judgement.

**Comment by henrikstranneheim** [comment]
Demultiplexing have moved to github actions - let's develop there instead

**Pull request closed without merging**